src/components/AssetTxHistory/AssetTxHistory.tsx

```diff
@@ -111,12 +99,15 @@ class AssetTxHistory extends React.Component<Props, State> {
                         timestamp={tx.timestamp!}
                     />
                 ))}
-                {countOfTxList > 0 && (
+                {txList.length > 0 && (
                     <div className="pagination-container">
                         <Pagination
                             activePage={this.state.activePage}
                             itemsCountPerPage={10}
-                            totalItemsCount={countOfTxList}
+                            totalItemsCount={
+                                (this.state.activePage - 1) * 10 +
+                                (txList.length + 1)
```

Comment:
It's clever!

Reply:
We should show the correct number if we reached the end.
Maybe we can make the indexer gives `hasMore` field in the response.

We may implement the `hasMore` feature like this. If a user requests `n` rows, query the DB `n+1` results. If the count of the result row is `n+1`, then there exists the next page.

Reply:
Thanks.

I will change it as soon as the API changes.